docs/main/home/introduction.mdx

```diff
@@ -78,11 +78,11 @@ Keep reading to know more about how Dyte works, or [**click here to jump to our
 
 <!-- ![Diagram explaining how Dyte works](./how-dyte-works.png) -->
 
-Dyte is based largely on the WebRTC protocol, and the features and optimizations we build on top of it. Although the protocol is supposed to be peer-to-peer, anyone who has tried implementing it by themselves would know that the implementions get real messy, real fast. To have a real working solution, servers with SFU, STUN, TURN, etc capabilities are required to optimize for user bandwidth and performance, route the data packets correctly and maintain state of certain client side configurations and preferences.
+Dyte is based largely on the WebRTC protocol, and the features and optimizations we build on top of it. Although the protocol is supposed to be peer-to-peer, anyone who has tried implementing it by themselves would know that the implementations get real messy, real fast. To have a real working solution, servers with SFU, STUN, TURN, etc capabilities are required to optimize user bandwidth and performance, route the data packets correctly and maintain a state of specific client-side configurations and preferences.
```

Comment:
Dyte is based largely on the WebRTC protocol, and the features and optimizations we build on top of it. Although the protocol is supposed to be peer-to-peer, anyone who has tried implementing it by themselves would know that the implementation gets real messy, real fast. To have a working solution, servers with SFU, STUN, TURN, etc. capabilities are required to optimize user bandwidth and performance, route the data packets correctly, and maintain the state of specific client-side configurations and preferences.